js/package.json

```diff
@@ -18,17 +18,17 @@
     "build": "lerna run build --ignore=token-entangler --ignore=candy-machine-ui",
     "build-web": "lerna run build --stream --scope @oyster/common --scope web",
     "start": "cross-env CI=true lerna run start --scope @oyster/common --stream --parallel --scope web",
-    "lint": "prettier -c 'packages/*/{src,test}/**/*.ts' && npm run lint:eslint",
-    "lint:eslint": "eslint 'packages/*/{src,test}/**/*.ts'",
-    "lint:fix": "prettier --write 'packages/*/{src,test}/**/*.{ts,tsx}' && eslint --fix 'packages/*/{src,test}/**/*.ts'",
+    "lint": "prettier -c 'packages/*/{src,test}/**/*.{ts,tsx}' && npm run lint:eslint",
```

Comment:
After landing this you're probably going to want to:

```
npx prettier --write -c 'packages/*/{src,test}/**/*.{ts,tsx}
```

…in a follow up commit.